### PR TITLE
Refactor handlers long files into modules

### LINKED ISSUE #51: Improve the structure of the project by converting long files into modules
# Improve structure and maintainability of the project by separating files into modules

This issue looks to address the tight coupling of some `event_handlers` and `handlers` that contain a lot of logic that is currently in one file to a module with each sub-function into a file. An example would be the [`./event_handlers/actions.py`](https://github.com/bermed28/bot-discord/blob/b514bb42be4049e29a98033fa558942ccc07ce37/event_handlers/actions.py) file which contains 7 pieces of logic that do not relate to one another:
- event_get_calendar
- event_get_curriculum
- event_telephone_guide
- event_parse_university_building
- event_help_menu
- generate_server_rules
- generate_faq

These could be separated into files under the directory `event_handlers/actions/` and be converted into a modulel. 

# Pros 🎨 
- Better maintainability 
- Better readability
- Logic decoupling

### PR DESCRIPTION
**Pull request by agSant01**
# Overview

This PR aims to improve code decoupling and strip long files into multiple files under a module. 

This PR only modifies files under the `handler/` directory.

## Benefits 🎉  
- Better readability  👓 
- Easier to onboard new developers 🆕 
- Better maintainability 🏗️ 

Closes #51

### CODE REVIEW

File: handlers/telephone_guide/faculty.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+//  Copyright © 2021 bermedDev. All rights reserved.
+//  Copyright © 2021 agSant01. All rights reserved.
+//  Copyright © 2021 teamMADE. All rights reserved.
+'''
+import discord
+
+_CSE_FACULTY = {
+    "Bienvenido Velez Rivera": "Acting Dean of Engineering\nFull Time Professor\nbienvenido.velez@upr.edu",
+    "Pedro I. Rivera Vega": "Acting CSE Director\nFull Time Professor\np.rivera@upr.edu",
+    "Manuel Rodriguez Martinez": "Associate Director\nFull Time Professor\nmanuel.rodriguez7@upr.edu",
```
Comment (bermed28) [inline handlers/telephone_guide/faculty.py:15]: Manuel ya no es Director Asociado

File: handlers/telephone_guide/faculty.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    "Wilson Rivera Gallego": "Full Time Professor\nwilson.riveragallego@upr.edu",
+    "Kejie Lu": "Full Time Professor\nkejie.lu@upr.edu",
+    "Heidy Sierra Gil": "Associate Professor\nheidy.sierra1@upr.edu",
+    "Emmanuel Arzuaga Cruz": "Full Time Professor\nearzuaga@ece.uprm.edu",
```
Comment (bermed28) [inline handlers/telephone_guide/faculty.py:19]: Arzuaga es Director Asociado

File: handlers/telephone_guide/faculty.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    "Marko Schütz Schmuck": "Full Time Professor\nmarko.schutz@upr.edu",
+    "Jose L. Melendez": "Special Assistant to the Chancellor\nFull Time Professor\njose.melendez37@upr.edu",
+    "Jaime Seguel": "Retired\njaime.seguel@upr.edu",
+    "Juan O. Lopez Gerena": "Instructor\njuano.lopez@upr.edu"
```
Comment (bermed28) [inline handlers/telephone_guide/faculty.py:23]: Juan Lopez ya no esta en RUM, es director del dept de CS en UPRA, needs to be removed

File: handlers/telephone_guide/faculty.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    "Bienvenido Velez Rivera": "Acting Dean of Engineering\nFull Time Professor\nbienvenido.velez@upr.edu",
+    "Pedro I. Rivera Vega": "Acting CSE Director\nFull Time Professor\np.rivera@upr.edu",
+    "Manuel Rodriguez Martinez": "Associate Director\nFull Time Professor\nmanuel.rodriguez7@upr.edu",
+    "Wilson Rivera Gallego": "Full Time Professor\nwilson.riveragallego@upr.edu",
+    "Kejie Lu": "Full Time Professor\nkejie.lu@upr.edu",
+    "Heidy Sierra Gil": "Associate Professor\nheidy.sierra1@upr.edu",
+    "Emmanuel Arzuaga Cruz": "Full Time Professor\nearzuaga@ece.uprm.edu",
+    "Marko Schütz Schmuck": "Full Time Professor\nmarko.schutz@upr.edu",
+    "Jose L. Melendez": "Special Assistant to the Chancellor\nFull Time Professor\njose.melendez37@upr.edu",
+    "Jaime Seguel": "Retired\njaime.seguel@upr.edu",
```
Comment (bermed28) [inline handlers/telephone_guide/faculty.py:22]: Retired, removed

File: handlers/telephone_guide/faculty.py [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    "Juan O. Lopez Gerena": "Instructor\njuano.lopez@upr.edu"
+}
+
+_ECE_FACULTY = {
+    "Gerson Beauchamp": "Full Time Professor\ngerson.beauchamp@upr.edu",
+    "Jaime Arbona Fazzi": "Full Time Professor\njaime.arbona@upr.edu",
```
Comment (bermed28) [inline handlers/telephone_guide/faculty.py:28]: Retired

**Review by bermed28 — CHANGES_REQUESTED** [review]

File: handlers/telephone_guide/faculty.py (hunk 4 above)
Comment (agSant01) [reply handlers/telephone_guide/faculty.py:22]: Cual linea de la 19-22?

File: handlers/telephone_guide/faculty.py (hunk 5 above)
Comment (agSant01) [reply handlers/telephone_guide/faculty.py:28]: Arbona?

**Review by bermed28 — APPROVED** [review]

**Pull request merged**